rootfs/templates/confd_settings.py

```diff
@@ -1,8 +1,11 @@
 import os
 
 # security keys and auth tokens
-SECRET_KEY = '{{ getv "/deis/controller/secretKey" }}'
-BUILDER_KEY = '{{ getv "/deis/controller/builderKey" }}'
+with open('/var/run/secrets/api/builder/auth/builder-key') as f:
+    BUILDER_KEY = f.read()
+
+with open('/var/run/secrets/api/django/secret-key') as f:
+    SECRET_KEY = f.read()
```

Comment:
Was mostly keeping consistency - `settings.py` is static, `confd_settings.py` is not. Could totally move to `settings.py` tho since it isn't happening as part of `confd`. I'm indifferent for now